### DISCUSSION TITLE
How do I update this thing? Whenever I press update_comfyui, it says to press any key, then it disappears.

### DISCUSSION THREAD

**Discussion by weirdlighthouse** [comfyanonymous/ComfyUI#1073] [Q&A] [answered]
that's it that's my question
![ttttttt](https://github.com/comfyanonymous/ComfyUI/assets/140368252/1b8b8be0-fb17-4e22-9d8a-f8c2d97fd88d)

**Comment by comfyanonymous** [comment, marked as answer]
The "Done!" means it updated.

**Reply by weirdlighthouse** [reply]
👍

**Reply by Ferniclestix** [reply]
dont forget to reload your browser window when you reload the server.  new stuff won't propagate until you reload it.
